plugins/ipc-rules/ipc-utility-methods.hpp

```diff
@@ -23,6 +23,8 @@ class ipc_rules_utility_methods_t
   private:
     wlr_backend *headless_backend = NULL;
     std::set<uint64_t> our_outputs;
+    int warning_count = 0;
```

Comment:
We should make these static variables in `set_config_options`, this way it is clear that they are for that IPC method only.